src/Microsoft.VisualBasic/tests/VB/FileIOTests.vb

```diff
@@ -0,0 +1,897 @@
+                DirectoryList = FileSystem.GetDirectories(TestBase.TestDirectory, SearchOption.SearchAllSubDirectories, "*000")
+                Assert.True(DirectoryList.Count = 2, $"Search results Expected 2 Actual {DirectoryList.Count} {DirectoryListToString(DirectoryList)}")
+            End Using
+        End Sub
+
+        <Fact>
+        Public Shared Sub GetDirectoryInfo_Directory()
+            Using TestBase As New FileIOTests
+                For i As Integer = 0 To 5
+                    IO.Directory.CreateDirectory(IO.Path.Combine(TestBase.TestDirectory, $"NewSubDirectory{i}"))
+                Next
+                IO.Directory.CreateDirectory(IO.Path.Combine(TestBase.TestDirectory, $"NewSubDirectory0", $"NewSubSubDirectory"))
+                Dim info As IO.DirectoryInfo = FileSystem.GetDirectoryInfo(TestBase.TestDirectory)
+                Dim infoFromIO As IO.DirectoryInfo = New IO.DirectoryInfo(TestBase.TestDirectory)
+                Assert.True(info.CreationTime = infoFromIO.CreationTime, $"Creation Time info ({info.CreationTime}) <> IO.DriveInfo.CreateTime {infoFromIO.CreationTime})")
+                Assert.True(info.Extension = infoFromIO.Extension, $"Extension {info.Extension} <> IO.DriveInfo.Extension {infoFromIO.Extension}")
+                Assert.True(info.FullName = TestBase.TestDirectory, $"Fullname {info.FullName} <> TestBase.TestDirectory {TestBase.TestDirectory}")
+                Assert.True(info.LastAccessTime = infoFromIO.LastAccessTime)
+                Assert.True(info.Name = infoFromIO.Name, $"Name {info.Name} Doesn't match {infoFromIO.Name}")
+                Assert.True(info.Parent.ToString = infoFromIO.Parent.ToString, "info.Parent<>infoFromIO.Parent")
+                Assert.True(info.Root.Name = infoFromIO.Root.Name, $"info.Root.Name {info.Root.Name} <> IO.DriveInfo.Root.Name){infoFromIO.Root.Name}")
+            End Using
+        End Sub
+
+        <Fact>
+        Public Shared Sub GetDriveInfo_Drive()
+            Dim Drives() As IO.DriveInfo = IO.DriveInfo.GetDrives()
+            Assert.True(Drives.Count > 0)
+            Assert.Equal(FileSystem.GetDriveInfo(Drives(0).Name).Name, New System.IO.DriveInfo(Drives(0).Name).Name)
+        End Sub
+
+        <Fact>
+        Public Shared Sub GetFileInfo_File()
+            Using TestBase As New FileIOTests
+                Dim TestFile As String = CreateTestFile(TestBase, SourceData)
+
+                Dim FileInfoFromSystemIO As IO.FileInfo = New IO.FileInfo(TestFile)
+                Assert.NotNull(FileInfoFromSystemIO)
+
+                Dim info As IO.FileInfo = FileSystem.GetFileInfo(TestFile)
+                Assert.NotNull(info)
+                With FileInfoFromSystemIO
+                    Assert.True(info.Exists)
+                    Assert.True(info.Attributes = .Attributes)
+                    Assert.True(info.CreationTime > Date.MinValue, "Starttime = 0")
```

Comment:
Is it necessary to compare this file differently, or could we use `info.CreationTime = .CreationTime` instead?

Reply:
I add the test you recommend but I also want to make sure Info.CreateTIme <> 0, this was a test I found elsewhere that I felt should be run once.